### DISCUSSION TITLE
Know if asset is a Live photo directly

### DISCUSSION THREAD

**Discussion by Tom3652** [fluttercandies/flutter_photo_manager#710] [Q&A] [answered]
Hello and thanks for supporting Live photos as well :) 

I would like to know if there was a way to know directly on the ```AssetEntity``` (such as `entity.type`) if it's a Live photo or not (without await) ? 

Or should we request the `entity.fileWithSubtype` and if it's null it's not a Live photo ? 
I saw that we could filter only LivePhotos so i wondered if this would be possible to add something like ```AssetType.livePhoto``` for example when any asset would be fetched ?

**Comment by AlexV525** [comment, marked as answer]
Hmm, did you miss something? https://github.com/fluttercandies/flutter_photo_manager/blob/0545a02b2399e66530caf1e066c6b24444bd2e34/lib/src/types/entity.dart#L359

**Reply by Tom3652** [reply]
haha excellent, sorry indeed i missed it 😅
